### ISSUE TITLE
Phoenix Mob

### ISSUE THREAD

**Issue by FireHDn** [AlexModGuy/Ice-and-Fire-Suggestions#1294]
The Phoenix, a magnificent creature, living in mythical forests. Phoenixes, when killed, will be reborn stronger, then before. They sometimes lay eggs, which can be stolen, but beware: If the Phoenix catches you doing that, your fate is sealed. The Eggs can be hatched in a blazing flame (throwing 3 blaze powder in fire), when hatched, the Phoenix will be wild. The Player needs to gain its trust by feeding it and co. Once you gained enough trust, he will serve you. The more he trusts you, the more he will be able to bring you. Once in a while he will drop Phoenix Feathers which can be used to craft a legendary weapon, by combining it in a smithing table with a Sword, it will be able to shoot fire by swinging it in air. I will Hope the devs will find inspiration from this post :D

**Comment by CTH999** [comment]
yeah! And, maybe they will (sometimes) heal players (Phoenix tears have healing powers), and if something "kills" they player (They aren't on a loaded world for over x days, (so, servers), it will mourn the player, and fly off? (It returns, jubilantly, if the player "survived" (returned))

also, maybe they can get old, die, and come back?
